southpark-incremental-indexing/app.py

```diff
@@ -0,0 +1,112 @@
+__copyright__ = "Copyright (c) 2020 Jina AI Limited. All rights reserved."
+__license__ = "Apache-2.0"
+
+import os
+
+import click
+from jina.flow import Flow
+
+
+def config():
+    os.environ["JINA_DATA_FILE_1"] = os.environ.get(
+        "JINA_DATA_FILE_1", "data/character-lines-1.csv"
+    )
+    os.environ["JINA_DATA_FILE_2"] = os.environ.get(
```

Comment:
would be nice to migrate this and southpark to single quote

Reply:
good point. done (for this example). cleaning up double quotes throughout the others should be a separate PR, I think

Reply:
Sure